src/content/en/fundamentals/accessibility/index.md

```diff
@@ -1,8 +1,355 @@
 project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
+description: Improving accessibility for web pages
 
-{# wf_updated_on: 2015-09-08 #}
-{# wf_published_on: 2015-09-08 #}
+{# wf_review_required #}
+{# wf_updated_on: 2016-06-26 #}
+{# wf_published_on: 2016-06-26 #}
 
 # Accessibility {: .page-title }
 
+{% include "_shared/contributors/megginkearney.html" %}
+{% include "_shared/contributors/dgash.html" %}
+{% include "_shared/contributors/robdodson.html" %}
+
+
+This document set is a text-based version of part of the content covered in the
+<a href="https://www.udacity.com/course/web-accessibility--ud891"
```

Comment:
Oof.. yeah sorry, I think there's like a zillion of these from the original script translation. Fixed  the ones on this page at least